(bug) Dashboard - top bar: Icons are not aligned in hight

StanislavShevchenko opened the pull request:
# Description

top bar: Icons are not aligned in hight

About #832 

# How Has This Been Tested?

go to dashboard, look on the top bar

# Checklist:
- [x] PR title matches follow: (Feature|Bug|Chore) Task Name
- [x] My code follows the style guidelines of this project
- [x] I have followed all the instructions described in the initial task (check Definitions of Done)
- [x] I have performed a self-review of my own code
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] I have added reference to a related issue in the repository
- [x] I have added a detailed description of the changes proposed in the pull request. I am as descriptive as possible, assisting reviewers as much as possible.
- [x] I have added screenshots related to my pull request ( for frontend tasks)
- [ ] I have pasted a gif showing the feature.
- [ ] @mentions of the person or team responsible for reviewing proposed changes
![image](https://user-images.githubusercontent.com/8187382/68373248-28636900-014b-11ea-8d6b-df95ed698d41.png)
![image](https://user-images.githubusercontent.com/8187382/68373333-58ab0780-014b-11ea-9c90-c442ba2c0c30.png)

sirpy commented on src/components/appNavigation/TabsView.js:
```diff
@@ -70,7 +70,7 @@ const TabsView = (props: TabViewProps) => {
       <Appbar.Content />
       {config.market && (
-        <TouchableOpacity onPress={goToMarketplace}>
+        <TouchableOpacity onPress={goToMarketplace} style={{ marginBottom: '5px', height: 53 }}>
```
```suggestion
        <TouchableOpacity onPress={goToMarketplace} style={{ marginBottom: 5, height: 53 }}>
```

sirpy commented:
@yaroslav-fedyshyn-nordwhale @StanislavShevchenko need to test after merge with marketbackground pr

yaroslav-fedyshyn-nordwhale commented:
I tested the latest changes (after merge) locally - the background for marketplace icon is broken.
<img width="475" alt="1" src="https://user-images.githubusercontent.com/49862004/68469221-6f705d80-0222-11ea-8455-b7040b2269d3.png">

StanislavShevchenko commented:
> I tested the latest changes (after merge) locally - the background for marketplace icon is broken.

i fixed this

yaroslav-fedyshyn-nordwhale commented:
> i fixed this

The market icon a bit higher than rewards.

yaroslav-fedyshyn-nordwhale commented:
> > i fixed this
> 
> The market icon a bit bigger than rewards. See the top line

StanislavShevchenko commented:
> > i fixed this
> 
> The market icon a bit higher than rewards.

fixed , I added height for icon
![image](https://user-images.githubusercontent.com/8187382/68471065-14406a00-0226-11ea-808d-95a8d8d93abe.png)
![image](https://user-images.githubusercontent.com/8187382/68471110-29b59400-0226-11ea-8674-1815801565e4.png)

yaroslav-fedyshyn-nordwhale commented:
> > > i fixed this
> > 
> > 
> > The market icon a bit higher than rewards.
> 
> fixed , I added height for icon

thanks
